(node:2979066) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-nosession-40529',
  containerIp: '10.0.3.137',
  mitmPort: 40529,
  autoAccessFixed: 1
}
2024-11-06 06:28:13.694 - [Info] MITM Version: 2
2024-11-06 06:28:13.696 - [Info] Auto Access Enabled: true
2024-11-06 06:28:13.697 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-06 06:28:13.713 - [Info] SSH man-in-the-middle server for 10.0.3.137 listening on 10.0.3.1:40529
2024-11-06 06:29:14.675 - [Debug] [Connection] Attacker connected: 8.222.128.215 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 06:29:16.154 - [Debug] [Auth] Attacker 8.222.128.215 trying to authenticate with 'password'
2024-11-06 06:29:16.155 - [Debug] [Auto Access] Attacker: 8.222.128.215, Threshold: 1, Attempts: 1
2024-11-06 06:29:16.156 - [Debug] [Auto Access] Compromising the honeypot
2024-11-06 06:29:16.156 - [Debug] [Auto Access] Adding the following credentials: 'vf:vf'
2024-11-06 06:29:16.980 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-06 06:29:16.982 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-06 06:29:16.983 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.137
2024-11-06 06:29:17.095 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-06 06:29:18.056 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-06 06:29:19.622 - [Debug] [Connection] Attacker connected: 8.222.128.215 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 06:29:21.236 - [Debug] [Auth] Attacker 8.222.128.215 trying to authenticate with 'password'
2024-11-06 06:29:21.236 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-06 06:29:21.247 - [Debug] [Auth] Attacker: 8.222.128.215 has so far made 1 attempts. Remaining: 5 attempts
2024-11-06 06:29:21.754 - [Debug] [Connection] Attacker closed the connection
2024-11-06 06:29:21.998 - [Debug] [Connection] Attacker connected: 8.222.128.215 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 06:29:23.507 - [Debug] [Auth] Attacker 8.222.128.215 trying to authenticate with 'password'
2024-11-06 06:29:23.508 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-06 06:29:23.508 - [Debug] [Auth] Attacker: 8.222.128.215 has so far made 1 attempts. Remaining: 5 attempts
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
